docs/develop/automatically-open-a-task-pane-with-a-document.md

```diff
@@ -20,9 +20,9 @@ With the autoopen feature, you can explicitly define or allow the user to define
 
```

Comment:
nit: In line 13, we mention the AddInCommands 1.1 requirement set. I think it would be useful if we link to that page. I'm not sure what the actual relative path is but probably something like ../reference/requirement-sets/add-in-commands-requirement-sets.md

Reply:
Done